docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -531,6 +531,116 @@ When the batch-consumer mode is enabled, the specific headers of batched message
 
 NOTE: When publish messages, all the above batch headers if exist will be removed from the messages to send.
 
+===== Multi Binder Support
+This sample takes connection string as example. Credentials of service priciple and MSI are also supported, users can set related properties in each binder's environment settings.
```

Comment:
Let's add more detailed introduction here: "Connection to mutiple Event Hubs namespaces is also supported by using multi binders."